roles/server_monitoring/templates/docker-compose.yml.j2

```diff
@@ -0,0 +1,109 @@
+version: '3'
+
+services:
+{% if server_monitoring_certbot_enabled %}
+  certbot:
+    image: {{ server_monitoring_certbot_image }}
+    container_name: certbot
+    restart: unless-stopped
+    user: {{ server_monitoring_certbot_user_id }}:{{ server_monitoring_certbot_group_id }}
+    volumes:
+      - {{ server_monitoring_host_letsencrypt_folder }}:/etc/letsencrypt
+      - {{ server_monitoring_docker_volumes_folder }}/webroot:/var/www/certbot
+    entrypoint: "/bin/sh -c 'trap exit TERM; while :; do certbot renew --logs-dir /certbot/logs --work-dir /certbot/work; sleep 12h & wait $${!}; done;'"
```

Comment:
Maybe a functionality to generate/configure (not only renew) the certificates could be integrated, if we want the role to be more "standalone",